src/PresentationalComponents/Charts/ActionsOverviewDonut.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { ChartDonut, ChartLabel, ChartLegend, ChartTheme } from '@patternfly/react-charts';
+import { Grid, GridItem } from '@patternfly/react-core';
+
+class AdvisorOverviewDonut extends React.Component {
+    getChart = (theme, typeNames) => (
+        <ChartDonut
+            data={ this.props.category.map((value, key) => ({ x: typeNames[key], y: value, label: `${typeNames[key]}: ${value}` })) }
+            theme={ theme }
+            height={ 200 }
+            width={ 200 }
+        />
+    );
+
+    getLegend = (theme, typeNames) => (
+        <ChartLegend
+            data={ this.props.category.map((value, key) => ({ name: `${typeNames[key]} (${value})` })) }
+            itemsPerRow={ 2 }
+            events={ [
+                {
+                    target: 'labels', eventHandlers: { onClick: this.legendClick }
+                }] }
+            theme={ theme }
+        />
+    );
+
+    legendClick = () => {
+        return [{
+            target: 'labels',
+            mutation: (props) => {
+                this.props.history.push(`/actions/${props.datum.name.split(' ')[0].toLocaleLowerCase()}`);
+            }
+        }];
+    };
+
+    render () {
+        const label = (
+            <div
+                className="chart-label"
+            >
+                <ChartLabel
+                    style={ { fontSize: 20 } }
+                    text={ this.props.category.length ? `${this.props.category.reduce((sum, curr) => sum + curr)} Issues` : '' }
+                    textAnchor="middle"
+                    verticalAnchor="middle"
+                />
+
+            </div>
+        );
+        const typeNames = [ 'Availability', 'Security', 'Stability', 'Performance' ];
+
+        return (
+            <Grid gutter="lg">
+                <GridItem lg={ 1 }/>
```

Comment:
What's the purpose of using these empty grid items?

Reply:
🤷‍♂️ make it kinda be in the center kinda, else it gets shoved over to the right (i'd not believe it unless i didn't see it 😭 )

Reply:
@AllenBW could you use the bullseye or just` margin-left/right: auto`?

Reply:
well it was a good thought.... bullseye does this
<img width="675" alt="screen shot 2018-12-13 at 6 44 56 am" src="https://user-images.githubusercontent.com/6640236/49936715-b0d97b00-fea2-11e8-8789-7e0ad26f4a5d.png">

as for adding css 😬